src/test/compile-fail/issue-13352.rs

```diff
@@ -23,4 +23,6 @@ fn main() {
         unsafe { libc::exit(0 as libc::c_int); }
     });
     2_usize + (loop {});
+    //~^ ERROR E0277
```

Comment:
(this was supposed to be a comment on commit 
f8a658c ; maybe its resolved in follow-on commits.)